tests/Behat/FeatureContext.php

```diff
@@ -199,4 +199,33 @@ public function theNodeShouldHaveTheFollowingVersion(string $title, TableNode $o
     Assert::assertEquals($options->getRowsHash(), $node_version_value);
   }
 
+  /**
+   * Check link to target.
+   *
+   * @param string $link
+   *   Link identifier.
+   * @param string $path
+   *   Target path of the link.
+   *
+   * @throws \Exception
+   *   Throws an exception if the link is not found or if the target is wrong.
+   *
+   * @Then I should see the link :link point to :path
```

Comment:
Is this possible to reuse this function inside oe_content lib
\Drupal\Tests\oe_content\Behat\FeatureContext::assertLinkWithHref
?

Reply:
We can, let's see if it doesn't clash for other tests.

Reply:
This too.

Reply:
But oe_content doesn't have any direct dependency with oe_editorial (nor do we have it the other way around), so how am I supposed to use this function?

Reply:
Yes I agree doubts of Imanol. Don't sure that it make sense add additional dependencies from oe_content.